examples/README.md

```diff
@@ -25,6 +25,12 @@ Available examples include:
 - **Flutter gallery** The [flutter gallery app](flutter_gallery) showcases
   Flutter's implementation of [material design](https://material.google.com/).
 
+- **Platform Services** The [platform services app](platform_services)
+  demonstrates how to connect a Flutter app to platform-specific services.
```

Comment:
I would s/platform-specific services/platform-specific APIs/.

And maybe add "For documentation, see https://flutter.io/platform-channels/".